lib/type/integer.ex

```diff
@@ -7,3 +7,12 @@ end
 defmodule Antikythera.MilliSecondsSinceEpoch do
   use Croma.SubtypeOfInt, min: 0
 end
+
+defmodule Antikythera.MilliSecondsInGregorian do
+  @time_epoch_offset_milliseconds (:calendar.datetime_to_gregorian_seconds({{1970, 1, 1}, {0, 0, 0}}) * 1000)
+  def time_epoch_offset_milliseconds(), do: @time_epoch_offset_milliseconds
+
+  # This restriction is temporarily expanded for backward compatibility.
+  # See ticket #177930 for details.
```

Comment:
Our Redmine is not public.
Therefore, please remove this comment.
(Commit message can include a ticket number, but we would like to avoid to write ticket number in codes)

Reply:
26c57136505c5872488028f343363d9efefe66bf